spring-cloud-dataflow-classic-docs/src/test/java/org/springframework/cloud/dataflow/server/rest/documentation/ApiDocumentation.java

```diff
@@ -234,6 +235,9 @@ public void index() throws Exception {
 
 						fieldWithPath("_links.tasks/thinexecutions.href").description("Link to the tasks/thinexecutions"),
 
+						fieldWithPath("_links.tasks/thinexecutions/name.href").description("Link to the tasks/thinexecutions/name"),
+						fieldWithPath("_links.tasks/thinexecutions/name.templated").type(JsonFieldType.BOOLEAN).optional().description("Link to the tasks/thinexecutions/name is templated"),
```

Comment:
The optional makes sense but I noticed it was not in the source being ported ([here](https://github.com/spring-cloud/spring-cloud-dataflow/pull/5994/files#diff-9158f2aa24bd9b317120f16404df0a58043b47e3479e147777f71dba6e36cba8R225)).

Reply:
Do you mind updating main after this w/ the same?

Reply:
Done